src/studio/src/designer/frontend/dashboard/features/serviceOverview/serviceCard.tsx

```diff
@@ -1,25 +1,18 @@
-import { Card, CardActionArea, CardContent, Grid, Typography } from '@material-ui/core';
-import { withStyles } from '@material-ui/core/styles';
+import { Card, CardActionArea, CardContent, Grid, IconButton, Typography, makeStyles } from '@material-ui/core';
 import classNames from 'classnames';
 import * as moment from 'moment';
 import * as React from 'react';
-import { connect } from 'react-redux';
 import TruncateMarkup from 'react-truncate-markup';
 import { getLanguageFromKey } from 'app-shared/utils/language';
+import { useSelector } from 'react-redux';
+import { IRepository } from 'app-shared/types';
+import ServiceMenu from './serviceMenu';
 
-export interface IServiceCardComponentProvidedProps {
-  classes: any;
-  service: any;
+export interface IServiceCardProps {
```

Comment:
where are we on moving away from the term service frontend? Would Repository be an okay substitution, and if so would this be the right time to introduce it?

Reply:
Since the dashboard is to be replaced sometime in the future I was hesitant to refactor as much in this code. I guess renaming is that much work, but unsure what real value it provides.